### ISSUE TITLE
Messy error if unattended setup wrongly installs first node as second node

### ISSUE THREAD

**Issue by kinvaris** [openvstorage/framework#910]
During setup of `openvstorage-backend` we've seen this error when providing a wrong `master-ip`:

```
root@ctrl-roub-01:~# ovs setup
+++++++++++++++++++++++++++++
+++  Open vStorage Setup  +++
+++++++++++++++++++++++++++++

+++ Setting up connections +++


+++ Collecting cluster information +++

Avahi installed
ERROR: Error loading storagerouters: [Errno 2] No such file or directory: '/opt/OpenvStorage/config/framework.json'

========= Remote Traceback (1) =========
Traceback (most recent call last):
  File "/tmp/tmp.I2OsXYmxlb/rpyc/core/protocol.py", line 305, in _dispatch_request
    res = self._HANDLERS[handler](self, *args)
  File "/tmp/tmp.I2OsXYmxlb/rpyc/core/protocol.py", line 535, in _handle_call
    return self._local_objects[oid](*args, **dict(kwargs))
  File "/opt/OpenvStorage/ovs/dal/lists/storagerouterlist.py", line 35, in get_storagerouters
    'items': []})
  File "/opt/OpenvStorage/ovs/dal/datalist.py", line 82, in __init__
    self._volatile = VolatileFactory.get_client()
  File "/opt/OpenvStorage/ovs/extensions/storage/volatilefactory.py", line 38, in get_client
    client_type = Configuration.get('/ovs/framework/stores|volatile')
  File "/opt/OpenvStorage/ovs/extensions/generic/configuration.py", line 112, in get
    data = Configuration._get(key_entries[0], raw)
  File "/opt/OpenvStorage/ovs/extensions/generic/configuration.py", line 362, in _get
    key=key)
  File "/opt/OpenvStorage/ovs/extensions/generic/configuration.py", line 390, in _passthrough
    store = Configuration.get_store()
  File "/opt/OpenvStorage/ovs/extensions/generic/configuration.py", line 419, in get_store
    with open(Configuration.BOOTSTRAP_CONFIG_LOCATION) as config_file:
IOError: [Errno 2] No such file or directory: '/opt/OpenvStorage/config/framework.json'



++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+++  An unexpected error occurred:                       +++
+++  Incorrect master IP provided, please choose from:   +++
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
```

**Comment by khenderick** [comment]
While this was a user mistake (first node was being set up as a second node, joining itself), we can use this ticket to clean up the error message a bit.

**Comment by khenderick** [comment]
Fixed by #1056, packaged in openvstorage-2.7.4-rev.4166.b187912

**Comment by wimpers** [comment]
Closed by PM.
